torch/csrc/jit/tracer.cpp

```diff
@@ -477,6 +477,14 @@ void addInputs(
     n->addInput(none);
   }
 }
+#ifdef BUILD_NAMEDTENSOR
+void addInputs(
+    Node* n,
+    const char* name,
+    c10::optional<at::DimnameList> value) {
+  TORCH_CHECK(false, "NYI: Named tensors are not supported with the tracer");
```

Comment:
Good catch. Yes, that'll be the case until I figure out JIT support for named tensors. Until then, all named tensor APIs must have different schemas from non-named tensor APIs so that the JIT is able to understand the non-named tensor APIs.